.github/workflows/build-and-deploy.yml

```
@@ -35,42 +35,33 @@ jobs:
           python3 -m pip install --upgrade pip
           pip install -r requirements.txt
 
-      - name: Generate Merged Data
-        run: |
-          python3 gh-data.py
-          python3 merge-data.py
-
-      - name: Switch to `gh-pages` and Preserve Changes
+      - name: Fetch and Check Out `gh-pages`
         run: |
           git config --global user.name "github-actions[bot]"
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
 
-          # Stash changes to preserve them
-          git stash --include-untracked
-
           # Fetch and switch to `gh-pages`
           git fetch origin gh-pages:gh-pages || git checkout --orphan gh-pages
```

I think that you can rely on gh-pages being there.

```suggestion
          git fetch origin gh-pages:gh-pages
```